### PR TITLE
Create advanced.html

### PR DESCRIPTION
**Pull request by jaganlal**
Another example to show how to 
1. set all brain tree fields (including business address)
2. if you already have a  client token

### CODE REVIEW

**Comment by jaganlal** [comment]
My project had a requirement where i had to do a POST to get the client token from my server. Due to this i had to change the way how client token is provided to braintree api ($braintree.api.Client). Please review my code and pass on your review comments

File: advanced.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          this.payButtonClicked = function() {
+
+            // - Validate $scope.creditCard
+            // - Make sure client is ready to use
+
+            client.tokenizeCard({
+              number: this.creditCard.number,
+              expirationDate: this.creditCard.expirationDate, 
+              expirationMonth: '04',
+              expirationYear: '2016',
```
Comment (jeffcarp) [inline advanced.html]: This probably shouldn't be hard-coded

File: advanced.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          });
```
Comment (jeffcarp) [inline advanced.html]: Hard-coding a client token will eventually stop working when the client token expires. It'd be better to use `server.js`.

File: advanced.html (hunk 1 above)
Comment (jeffcarp) [inline advanced.html]: Given the data above, looks like lines 50 and 51 should just be removed.

**Comment by jeffcarp** [comment]
Thanks for the contribution! Left a few comments. Also, haven't checked why the build is failing but that should get resolved before merge.

File: advanced.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          this.payButtonClicked = function() {
+
+            // - Validate $scope.creditCard
+            // - Make sure client is ready to use
+
+            client.tokenizeCard({
+              number: this.creditCard.number,
+              expirationDate: this.creditCard.expirationDate, 
+              expirationMonth: ('0'+(new Date().getMonth()+1)).slice(-2),
+              expirationYear: new Date().getFullYear() + '',
```
Comment (jeffcarp) [inline advanced.html]: Sorry! I might not have been super clear 😞  You can only pass `expirationDate` or `expiration{Month,Year}`, not both.

**Comment by jeffcarp** [comment]
I appreciate your contribution, but I'm closing this PR due to inactivity (so I don't lose my mind in a sea of issues). Please feel free to re-open to address comments.

**Pull request closed without merging**